game/definitions.rpy

```diff
@@ -751,7 +755,54 @@ init -999 python in utils:
         """
         return pprint.pformat(object.__dict__, indent, width)
 
+    def get_mouse_position():
+        """
+        Returns a tuple representing the mouse's current position in the game window.
+
+        OUT:
+            - mouse position as a tuple in format (x,y)
+        """
+        return pygame.mouse.get_pos()
+
+    def validate_key():
```

Comment:
Make this private too. It's easy enough to override this with a func that just returns true

Reply:
Done